chia/timelord/timelord_launcher.py

```diff
@@ -88,8 +88,11 @@ async def spawn_process(
             dirname = path_to_vdf_client.parent
             basename = path_to_vdf_client.name
             resolved = await resolve(host, prefer_ipv6=prefer_ipv6)
-            proc = await asyncio.create_subprocess_shell(
-                f"{basename} {resolved} {port} {counter}",
```

Comment:
yikes! you better hope none of those strings have spaces in them!

Reply:
Quite so...  All the more reason to not shove a shell in the middle.